### DISCUSSION TITLE
删除.umirc 文件 在 /config/config.ts配置项目无法运行

### DISCUSSION THREAD

**Discussion by Ark6Rj** [umijs/umi#12173] [Q&A]
删除了 .umirc 文件把内容放 在 src/config/config.ts 里面配置结果报错是为啥
![image](https://github.com/umijs/umi/assets/59910643/62e1f071-caac-4034-b37a-e6f99b8dd75b)

**Comment by fz6m** [comment]
如果想把配置文件放到 config 里，需要注意原配置文件 `.umirc.ts` 里的内容有没有涉及路径相关的，如果没有，就剪切移动到 `config/config.ts` 就可以，不是 `/config/config.ts` 。

如有问题，需要给一个最小复现。
